### PR TITLE
Use booleans in map rendering condition

### LINKED ISSUE #363: error: map cannot be displayed
@ptbrowne 
**Describe the bug**
Unexpected error
An unexpected error occurred while displaying this chart.

**To Reproduce**
1. Go to https://test.visualize.admin.ch/v/I2WxBXeQt2JY

**Expected behavior**
display map

**Environment (please complete the following information):**
 - Visualize environment and version: TEST v3.3.0 ([a3dcdaf](https://github.com/visualize-admin/visualization-tool/commit/a3dcdafab218e9bdee34ae3d7d3b7babb0e2e9ea))

### PR DESCRIPTION
**Pull request by bprusinowski**
Fixes #363.

Changes the logic so the booleans are used for map rendering condition.

### CODE REVIEW

File: app/charts/map/map.tsx [hunk 1]
```diff
@@ -208,152 +208,150 @@ export const MapComponent = () => {
...
-                        type: "INTERACTION_UPDATE",
-                        value: {
-                          interaction: {
-                            visible: true,
-                            mouse: { x, y },
-                            d: object.properties.observation,
-                          },
+      {(areaLayer.data.length > 0
+        ? shapes.features !== undefined
+        : features.symbolLayer?.points.length !== undefined) && (
```
Comment (ptbrowne) [inline app/charts/map/map.tsx:213]: Nit: This condition is not super clear, maybe there could be temp variables like `hasPoints`, `hasShapes`  ?

**Review by ptbrowne — COMMENTED** [review]
If it works, cool, but I think its not super readable there, could it be improved a bit ?

File: app/charts/map/map.tsx (hunk 1 above)
Comment (bprusinowski) [reply app/charts/map/map.tsx]: Thanks @ptbrowne, you were right it wasn't very clear; I refactored some code in preceding steps, hopefully it is be better now :)

**Review by ptbrowne — APPROVED** [review]
LGTM ! Thanks !

**Pull request merged**